shell/platform/darwin/macos/framework/Source/FlutterIOSurfaceHolder.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+
+/**
+ * FlutterIOSurfaceHolder maintains an IOSurface
+ * and provides an interface to bind the IOSurface to a texture.
+ */
+@interface FlutterIOSurfaceHolder : NSObject
+
+/**
+ * Bind the IOSurface to the provided texture and fbo.
+ */
+- (void)bindSurfaceToTexture:(GLuint)texture fbo:(GLuint)fbo size:(CGSize)size;
+
+/**
+ * Releases the current IOSurface if one exists
+ * and creates a new IOSurface with the specified size.
+ */
+- (void)createOrReplaceIOSurface:(CGSize)size;
+
+/**
+ * Returns a reference to the underlying IOSurface.
+ */
+- (_Nonnull const IOSurfaceRef&)getIOSurface;
```

Comment:
use `nonnull` instead of `_Nonnull` everywhere. Also `ioSurface` is more idiomatic obj-c than `getIOSurface`.

Reply:
Removed nonnull altogether, I don't think it's necessary for reference type here, correct me if I'm wrong.

Changed getIOSurface -> ioSurface